core/src/main/java/org/libdohj/params/AbstractNamecoinParams.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright 2013 Google Inc.
```

Comment:
I think given how much of this is new, it should have you listed for copyright. I think Google would mostly look confused at this point!